### PR TITLE
Evknorab1

### PR DESCRIPTION
**Pull request by ghost**
Updated EVK-NORA-B10 and added EVK-NORA-B12 for NCS v2.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,11 @@ GitHub repository.
 
 | EVK board     | Folder                    | Remarks                                 |
 |---------------|---------------------------|-----------------------------------------|
-| EVK-ANNA-B4   | ubx_evkannab4_nrf52833    | Tested with NCS v2.2.0 / Zephyr 3.2.99 |
-| EVK-NORA-B10  | ubx_evknorab10_nrf5340    | Tested with NCS v2.0.x / Zephyr 3.0.99  |
+
+| EVK-ANNA-B4   | ubx_evkannab4_nrf52833    | Tested with NCS v1.6.1 / Zephyr 2.6-rc1 |
```
Comment (perssonmagnus) [inline README.md]: There was actually an update for the EVK-ANNA-B4 where the config was reduced and tested with 2.2.0.
Plan is to extend this when time permits.

**Comment by dsconyers** [comment]
Should the EVK-NORA-B12 BSP include setting CONFIG_BT_CTLR_TX_PWR_ANTENNA=13?

**Comment by ghost** [comment]
It’s in zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpunet_defconfig


Bob Recny

Senior Engineer

u-blox San Diego, Inc.

2601 25th St SE Ste 200

Salem, OR 97302-1281

USA


Phone +1 971 231 8580

Mobile +1 503 423 7376

www.u-blox.com<https://www.u-blox.com/>

Reliable. Smart. Secure.

join our community<https://portal.u-blox.com/> | subscribe to our blog<https://www.u-blox.com/en/blog>
follow us on twitter<https://twitter.com/ublox> | subscribe to our youtube channel<https://www.youtube.com/c/ublox1> | follow us on linkedin<https://www.linkedin.com/company/u-blox> | connect with us on facebook<https://www.facebook.com/ublox>

This message contains confidential information intended only for the use of the addressee(s) and may be legally privileged. If you are not the intended recipient, you are hereby notified that any use, dissemination, or reproduction is strictly prohibited and may be unlawful. If you are not the intended recipient, please contact the sender by return email and destroy all copies of the original message.

<https://aka.ms/o0ukef>
________________________________

**Comment by dsconyers** [comment]
Thanks! I see it now.  I mistakenly inspected the .config file for the app core instead of the network core after making a build for the ubx_evknorab12.

File: zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpunet.dts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+		sw0 = &button0;
+		sw1 = &button1;
+		sw2 = &button2;
+		sw3 = &button3;
+		bootloader-led0 = &led0;
+		mcuboot-button0 = &button0;
+		mcuboot-led0 = &led0;
+		watchdog0 = &wdt0;
+	};
+// brec
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpunet.dts]: Remove //brec comment?

**Review by perssonmagnus — COMMENTED** [review]
Looks good as far as I can see.

File: zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpunet.dts (hunk 2 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpunet.dts]: Removed and replaced with `	/* Enable the NORA-B12 FEM */` in most recent commit.

**Review by perssonmagnus — APPROVED** [review]
Looks good to me.

**Pull request merged**